static/js/pages/applications/ApplicationDashboardPage.js

```diff
@@ -254,15 +254,17 @@ export class ApplicationDashboardPage extends React.Component<Props, State> {
             <div className="row">
               <div className="col-5" />
               <div className="col-3 view-statement">
-                <a
-                  className="btn-link"
-                  href={reverse(routes.applications.paymentHistory.self, {
-                    applicationId: application.id
-                  })}
-                >
-                  <span className="material-icons">printer</span>
-                  View Statement
-                </a>
+                {application.has_payments ? (
```

Comment:
Nit, take or leave: if theres just a single predicate, the ternary can be written like this...

```
{predicate && (
  <div>jsx</div>
)}
```